v4/source/brand.yaml

```diff
@@ -78,3 +78,30 @@
             }
 
             ok, resp := Client.UploadBrandImage(data.Bytes())
+
+    delete:
+      tags:
+        - brand
+      summary: Delete current brand image
+      description: |
+        Deletes the previously uploaded brand image. Returns 404 if no brand image has been uploaded.
+        ##### Permissions
+        Must have `manage_system` permission.
+      responses:
+        '200':
+          description: Brand image succesfully deleted
+          schema:
+            $ref: '#/definitions/StatusOK'
+        '404':
+          description: No brand image uploaded
+          $ref: '#/responses/NotFound'
```

Comment:
The server code appears to be able to return a `401` and `403` in some cases too.